go/adbc/driver/snowflake/driver_test.go

```diff
@@ -2253,3 +2253,44 @@ func TestJSONUnmarshal(t *testing.T) {
 		}
 	}
 }
+
+func (suite *SnowflakeTests) TestQueryTag() {
+	u, err := uuid.NewV7()
+	suite.Require().NoError(err)
+	tag := u.String()
+	suite.Require().NoError(suite.stmt.SetOption(driver.OptionStatementQueryTag, tag))
+
+	val, err := suite.stmt.(adbc.GetSetOptions).GetOption(driver.OptionStatementQueryTag)
+	suite.Require().NoError(err)
+	suite.Require().Equal(tag, val)
+
+	suite.Require().NoError(suite.stmt.SetSqlQuery("SELECT 1"))
+	rdr, n, err := suite.stmt.ExecuteQuery(suite.ctx)
+	suite.Require().NoError(err)
+	defer rdr.Release()
```

Comment:
i doubt it, but is there any way we can test / confirm that the tag is being propagated for the query?

Reply:
That _is_ the test: we query the query history with the tag and make sure it comes back

Reply:
Ah i get it. I missed that's what the query below was doing. Makes sense! Thanks!